### PR TITLE
Fix typos and formatting for swift cheatsheet

### PR DESCRIPTION
**Pull request by jmingtan**
https://duck.co/ia/view/swift_cheat_sheet
Maintainer: @gautamkrishnar

### CODE REVIEW

**Comment by daxtheduck** [comment]
@jmingtan Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by gautamkrishnar** [comment]
@jmingtan  check this https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2804#discussion_r58500439

**Comment by jmingtan** [comment]
@gautamkrishnar ok done

**Comment by moollaza** [comment]
@jmingtan thanks again for improving the Swift Cheat Sheet! You’ve been doing a really good job here and we'd like you to help our community tackle the next big challenge - our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements 

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Comment by gautamkrishnar** [comment]
@moollaza  i think this is good to merge.. :+1:

File: share/goodie/cheat_sheets/json/swift.json [hunk 1]
```diff
@@ -156,21 +156,21 @@
         "val": "Creating an Array with an Array Literal"
       },
       {
-        "key": "for item in shoppingList \\{\nprint(item)\n\\}",
+        "key": "for item in shoppingList \\{\n  print(item)\n\\}",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/swift.json]: @jmingtan @gautamkrishnar are these spaced meant to indent the text? If so, I would suggest using `\t` instead as we have the CheatSheet parser setup to automatically replace `\t` with two spaces ;)

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/swift.json]: okay noted that :+1:

**Comment by gautamkrishnar** [comment]
fixes:#2809

**Comment by moollaza** [comment]
@jmingtan Thanks! This LGTM :+1:

**Comment by zekiel** [comment]
@jmingtan we haven't gotten much Swift volunteership. Would you be interested in working on Swift IAs to help close the gap in unanswered swift queries? There's a table with some resource [here](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage) but full docs search would be a great start.

**Comment by jmingtan** [comment]
@zekiel thank you for the offer, I might take a look when I have time

**Comment by alohaas** [comment]
Hey @jmingtan if you are still interested in helping us with Swift IAs, I'd be happy to chat via Slack about how you can help :)

**Pull request merged**